services.md

```diff
@@ -34,8 +37,13 @@ manipulation of their state.
 A service communicates with others by exchanging state information about
 entities.
 
+#### What is a **micro** service?
+
+The **micro** in micro service applies to the number of domain concepts a service operates on. All services should endeavour to be small _in scope_, i.e. should only address one (or two) domain concepts and be **very** resistant to adding more.
```

Comment:
Maybe :) Let's not get too prescriptive. It may make sense (in some circumstances) to have a service that goes across multiple domain concepts.